charts/spire/charts/spire-server/README.md

```diff
@@ -129,17 +129,28 @@ A Helm chart to install the SPIRE server.
 | telemetry.prometheus.podMonitor.namespace | string | `""` | Override where to install the podMonitor, if not set will use the same namespace as the spire-server |
 | tolerations | list | `[]` |  |
 | topologySpreadConstraints | list | `[]` |  |
-| tornjak.config.dataStore | object | `{"driver":"sqlite3","file":"/run/spire/data/tornjak.sqlite3"}` | persistent DB for storing Tornjak specific information |
+| tornjak.config.dataStore | object | `{"driver":"sqlite3","file":"/run/spire/data/tornjak.sqlite3"}` | Persistent DB for storing Tornjak specific information |
+| tornjak.config.http.enabled | bool | `true` | Enables Tornjak HTTP (insecure) service |
+| tornjak.config.http.port | int | `10000` | Container port value for HTTP |
+| tornjak.config.http.service | object | `{"annotations":{},"port":10000,"type":"ClusterIP"}` | Service to handle Tornjak HTTP connection |
+| tornjak.config.mtls.ca | string | `"/opt/spire/sample-keys/rootCA.pem"` | CA for mTLS connection |
+| tornjak.config.mtls.cert | string | `"/opt/spire/sample-keys/cert.pem"` | Certificate for mTLS connection |
+| tornjak.config.mtls.enabled | bool | `false` | Enables Tornjak TLS service |
+| tornjak.config.mtls.key | string | `"/opt/spire/sample-keys/key.pem"` | Key for mTLS connection |
+| tornjak.config.mtls.port | int | `30000` | Container port value for mTLS |
+| tornjak.config.mtls.service | object | `{"annotations":{},"port":30000,"type":"ClusterIP"}` | Service to handle Tornjak mTLS connection |
+| tornjak.config.tls.cert | string | `"/opt/spire/sample-keys/cert.pem"` | Certificate for TLS connection |
+| tornjak.config.tls.enabled | bool | `false` | Enables Tornjak TLS service |
+| tornjak.config.tls.key | string | `"/opt/spire/sample-keys/key.pem"` | Key for TLS connection |
+| tornjak.config.tls.port | int | `20000` | Container port value for TLS |
+| tornjak.config.tls.service | object | `{"annotations":{},"port":20000,"type":"ClusterIP"}` | Service to handle Tornjak TLS connection |
```

Comment:
are http/mtls/tls exclusive or can you really enable more then one?

Reply:
You have to enable AT LEAST one, otherwise there is no point of running it, since you can't communicate, therefore if none is configured, Tornjak startup configuration validation throws an error

Reply:
Is there reason to have more then one though? Just wondering if this should be a mode enum rather then a set of bools.

Reply:
maybe an alternate angle... The mtls part, is that for user auth?

Reply:
As is, it can be used for user authentication, however I am preparing a next PR to bring a proper user authentication and authorization via Keycloak.

Reply:
BTW I liked your suggestion of separating the userCA into a second secret. It's done.